### ISSUE TITLE
Drag and Drop has no effect

### ISSUE THREAD

**Issue by zolrath** [RolandPheasant/TailBlazer#80]
When I attempt to drag and drop a file onto Tail Blazer end up with the :no_entry_sign: cursor.
I've tried a variety of file types and the results have been the same on all of them.

Running Windows 10 and I do have .NET 4.6.1 installed.

**Comment by RolandPheasant** [comment]
Can you open a file by pressing the open file button (one of the buttons on the title bar)

**Comment by zolrath** [comment]
Yes! I can path to it and everything performs as expected once open, it's only
the drag and drop functionality that doesn't work. On Fri, Feb 12, 2016 at 12:40 AM, Roland Pheasant < notifications@github.com [notifications@github.com] > wrote:
Can you open a file by pressing the open file button (one of the buttons on the
title bar)

—
Reply to this email directly or view it on GitHub
[https://github.com/RolandPheasant/TailBlazer/issues/80#issuecomment-183228683] .[https://github.com/notifications/beacon/AAbvoyj7kSBFLIedbXZ1YYtcQpDtVX4Jks5pjZHrgaJpZM4HY2o9.gif]

**Comment by RolandPheasant** [comment]
This is odd and I have never seen it before. Also drag and drop is code that was writen in week 1 of the project so I am perplexed.

**Comment by zolrath** [comment]
Another diagnostic point is that I'm using a Surface Book, if the presence of touch + pen input interferes with your drag and drop in some way?

**Comment by RolandPheasant** [comment]
That's definitely a clue. I will do some research

**Comment by alexandernyquist** [comment]
Could it be that you are running Tail Blazer as Administrator and the file is owned by someone else, or vice versa? I observe the same behaviour, navigating to the file works but drag and drop does not.

**Comment by zolrath** [comment]
Yes! If I launch Tail Blazer from an Administrator cmd prompt Drag and Drop doesn't work but non-Administrator did allow me to drag and drop.

**Comment by RolandPheasant** [comment]
Any ideas about how I work around this one?

**Comment by zolrath** [comment]
This article is quite old but seems to explain the the issue.
http://blogs.msdn.com/b/patricka/archive/2010/01/28/q-why-doesn-t-drag-and-drop-work-when-my-application-is-running-elevated-a-mandatory-integrity-control-and-uipi.aspx

It seems to be a security feature but you can explicitly opt-in to allowing specific messages through from lower-level processes.

**Comment by RolandPheasant** [comment]
@zolrath @alexandernyquist  interesting. I tried running TB as an administrator and sure enough drag and drop failed. As stated, I could use ChangeWindowsMessageFilterEx  but that is not recommended and frankly is sucks.  Need to have a think about this one...

**Comment by mizipzor** [comment]
I also have this. Found this bug report with the workaround, everything works here as well when I don't run as admin.
